coreos-base/oem-packet/files/reset-interfaces.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
I'm looking further into this. Just running that instead of what we have does not bring our bond configuration up properly. This could be a difference in how we're doing network over rackspace.